src/kajol/WhileLoopExample.java

```diff
@@ -0,0 +1,63 @@
+package kajol;
+
+public class WhileLoopExample {
+	
+	void printEvenNumbers(int start, int end) {
+		System.out.println("Even Numbers between "+start+" and "+end+" :");
+		int index=start;
```

Comment:
You can handle this scenario without creating third variable, think on this.
can you use start++ for increment?